ui/package.json

```diff
@@ -70,7 +70,7 @@
     "tailwindcss": "^3.4.1",
     "tailwindcss-animate": "^1.0.7",
     "typescript": "^5.2.2",
-    "vite": "^5.1.6",
+    "vite": "5.3.5",
```

Comment:
Nit: Should this better be `~5.3.5` to allow for patch level updates?

Reply:
I'd rather not do any automatic updates until we know what caused the issue. Also I think the Vite team does not maintain old minor versions anyway.